discovery-provider/src/api/v1/models/users.py

```diff
@@ -114,3 +114,17 @@
         "metadata": fields.Raw(required=True),
     },
 )
+
+decoded_user_token = ns.model(
+    "decoded_user_token",
+    {
+        "userId": fields.String(required=True),
```

Comment:
Generally, the convention used in discovery is to use `snake_case` for api responses vs `camelCase`, but not sure if there's some standard for camelcase in oauth flows.

Reply:
Yeah totally - I originally snake cased everything but then realized it would be confusing to change the key/property names from camel case (which is what the original JWT payload has) to snake case. And yeah camelCase seems to be the standard for JSON https://stackoverflow.com/questions/5543490/json-naming-convention-snake-case-camelcase-or-pascalcase